src/app.js

```diff
@@ -297,6 +298,18 @@ export const start = async args => {
     screen.spawn('git', ['log', args.join('/'), ...gitLogArguments]);
   });
 
+  branchTable.key('y', function() {
+    const selection = parseSelection(this.items[this.selected].content);
+    const gitBranch = selection[2];
+    copyToClipBoard(gitBranch)
+      .then(() => {
+        logger.log(`branch name ${gitBranch} copied.`);
```

Comment:
Can you capitalize the first character of this sentence?

Reply:
done.